### PR TITLE
fixed the issue: dropdown granite models are editable

### PR DESCRIPTION
**Pull request by lavanyaj3**
fixed the issue that "dropdown granite models are editable"

### CODE REVIEW

**Review by deboer-tim — APPROVED** [review]
LGTM

File: webviews/src/App.tsx [hunk 1]
```diff
@@ -268,7 +271,7 @@ function App() {
           status={getModelStatus(embeddingsModel)}
           options={embeddingsOptions}
           progress={embeddingsModel ? modelPullProgress[embeddingsModel] : undefined}
-          disabled={!enabled}
+          readOnly={true} // Added readOnly prop here
```
Comment (fbricon) [inline webviews/src/App.tsx:274]: I'm not sure this is correct, I need to check

Comment (lavanyaj3) [reply webviews/src/App.tsx:274]: yes please. Let me know if any changes are to be made.

Comment (fbricon) [reply webviews/src/App.tsx:274]: this makes model selection impossible

**Review by fbricon — CHANGES_REQUESTED** [review]
this whole PR should be a single line change L138:
```
isSearchable={false} // Disable typing in dropdown
```

**Comment by lavanyaj3** [comment]
will make the change accordingly

**Comment by lavanyaj3** [comment]
made the suggested change and reverted back all the other changes

**Comment by lavanyaj3** [comment]
cancelling this pull request. Will raise another with the suggested change

**Pull request closed without merging**